helm_tests/airflow_aux/test_airflow_common.py

```diff
@@ -408,3 +408,39 @@ def test_priority_class_name(self):
             priority = doc["spec"]["template"]["spec"]["priorityClassName"]
 
             assert priority == f"low-priority-{component}"
+
+    # Test containerLifecycleHooks for main containers
+    def test_global_container_lifecycle_webhooks(self):
+        post_start_value = {"exec": {"command": ["bash", "-c", "echo postStart"]}}
+        pre_stop_value = {"exec": {"command": ["bash", "-c", "echo preStop"]}}
+        docs = render_chart(
+            values={
+                "containerLifecycleHooks": {
+                    "postStart": post_start_value,
+                    "preStop": pre_stop_value,
+                },
+            },
+            show_only=[
+                "templates/flower/flower-deployment.yaml",
+                "templates/scheduler/scheduler-deployment.yaml",
+                "templates/webserver/webserver-deployment.yaml",
+                "templates/workers/worker-deployment.yaml",
+                "templates/jobs/create-user-job.yaml",
+                "templates/jobs/migrate-database-job.yaml",
+                "templates/triggerer/triggerer-deployment.yaml",
+                "templates/statsd/statsd-deployment.yaml",
+                "templates/redis/redis-statefulset.yaml",
+            ],
+        )
+
+        for index in range(len(docs) - 2):
```

Comment:
Can you refactor this to use list slices instead? e.g like [this](https://github.com/apache/airflow/blob/0fbb05751acf6061f9f6fa83ee53816d568055d0/helm_tests/security/test_security_context.py#L247).